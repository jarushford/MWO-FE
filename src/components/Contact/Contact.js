import React from 'react'
import '../../main.scss'

export default function Contact() {
  return (
    <section className="contact-page">
      <h1>contact</h1>
      <div className="contact-container">
        <h4>for booking and general inquiries email us at:</h4>
        <h3><a href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!">madwallaceband@gmail.com</a></h3>
        <button>View EPK</button>
      </div>
    </section>
  )
}