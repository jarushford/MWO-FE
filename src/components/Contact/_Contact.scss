.contact-container {
  width: 100%;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  align-items: center;

  h4 {
    color: white;
    font-size: 1.2rem;
    margin-top: 60px;
  }

  h3 {
    color: white;
    font-size: 1.8rem;
    margin-top: 0;
  }

  a, a:visited {
    color: $purple;
  }

  button {
    width: 140px;
    height: 45px;
    background: none;
    border: 2px solid $purple;
    border-radius: 5px;
    color: $purple;
    font-family: 'Open Sans', sans-serif;
    font-size: 1.3rem;
    transition: .3s ease;
    outline: none;
    margin-top: 50px;
  
    &:hover {
      background: $purple;
      color: white;
    }
  }
}

.contact-page {
  h1 {
    color: $lightOverlay;
    font-size: 15rem;
    margin: 0;
    opacity: .35;
    height: 175px;
    line-height: 150px;
  }
}