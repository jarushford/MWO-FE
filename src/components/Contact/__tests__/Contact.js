import React from 'react'
import { shallow } from 'enzyme'
import Contact from '../Contact'

describe('Contact', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Contact />)

    expect(wrapper).toMatchSnapshot()
  })
})