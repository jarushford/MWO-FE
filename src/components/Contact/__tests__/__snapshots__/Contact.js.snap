// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contact />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          contact
        </h1>,
        <div
          className="contact-container"
        >
          <h4>
            for booking, merch, and general inquiries email us at:
          </h4>
          <h3>
            <a
              href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!"
            >
              madwallaceband@gmail.com
            </a>
          </h3>
          <Link
            replace={false}
            to="/presskit"
          >
            <button>
              View EPK
            </button>
          </Link>
        </div>,
      ],
      "className": "contact-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "contact",
        },
        "ref": null,
        "rendered": "contact",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h4>
              for booking, merch, and general inquiries email us at:
            </h4>,
            <h3>
              <a
                href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!"
              >
                madwallaceband@gmail.com
              </a>
            </h3>,
            <Link
              replace={false}
              to="/presskit"
            >
              <button>
                View EPK
              </button>
            </Link>,
          ],
          "className": "contact-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "for booking, merch, and general inquiries email us at:",
            },
            "ref": null,
            "rendered": "for booking, merch, and general inquiries email us at:",
            "type": "h4",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!"
              >
                madwallaceband@gmail.com
              </a>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "madwallaceband@gmail.com",
                "href": "mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!",
              },
              "ref": null,
              "rendered": "madwallaceband@gmail.com",
              "type": "a",
            },
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <button>
                View EPK
              </button>,
              "replace": false,
              "to": "/presskit",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "View EPK",
              },
              "ref": null,
              "rendered": "View EPK",
              "type": "button",
            },
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            contact
          </h1>,
          <div
            className="contact-container"
          >
            <h4>
              for booking, merch, and general inquiries email us at:
            </h4>
            <h3>
              <a
                href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!"
              >
                madwallaceband@gmail.com
              </a>
            </h3>
            <Link
              replace={false}
              to="/presskit"
            >
              <button>
                View EPK
              </button>
            </Link>
          </div>,
        ],
        "className": "contact-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "contact",
          },
          "ref": null,
          "rendered": "contact",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h4>
                for booking, merch, and general inquiries email us at:
              </h4>,
              <h3>
                <a
                  href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!"
                >
                  madwallaceband@gmail.com
                </a>
              </h3>,
              <Link
                replace={false}
                to="/presskit"
              >
                <button>
                  View EPK
                </button>
              </Link>,
            ],
            "className": "contact-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "for booking, merch, and general inquiries email us at:",
              },
              "ref": null,
              "rendered": "for booking, merch, and general inquiries email us at:",
              "type": "h4",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!"
                >
                  madwallaceband@gmail.com
                </a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "madwallaceband@gmail.com",
                  "href": "mailto:madwallaceband@gmail.com?subject=Hello Mad Wallace!",
                },
                "ref": null,
                "rendered": "madwallaceband@gmail.com",
                "type": "a",
              },
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <button>
                  View EPK
                </button>,
                "replace": false,
                "to": "/presskit",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "View EPK",
                },
                "ref": null,
                "rendered": "View EPK",
                "type": "button",
              },
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
